doc/dev/tests.md

```diff
@@ -5,24 +5,42 @@ title: Tests
 
 ## Overview
 
-Npgsql comes with an extensive test suite to make sure no regressions occur. All tests are run on our build server on all supported .NET versions (including a recent version of mono) and all supported PostgreSQL backends.
+Npgsql has an extensive test suite to guard against regressions. The test suite is run on the official build server for all supported .NET versions (including a recent version of mono) and all supported PostgreSQL backends.
```

Comment:
As long as you're changing this text it may be nice to mention Appveyor and Travis, and even provide public links to them.

Reply:
Updated with links, let me know if the wording looks okay.